src/apps/testapps/testH3Index.c

```diff
@@ -68,19 +68,19 @@ SUITE(h3Index) {
 
     TEST(isValidCellAtResolution) {
         for (int i = 0; i <= MAX_H3_RES; i++) {
-            GeoPoint GeoPoint = {0, 0};
+            LatLng LatLng = {0, 0};
             H3Index h3;
-            t_assertSuccess(H3_EXPORT(pointToCell)(&GeoPoint, i, &h3));
+            t_assertSuccess(H3_EXPORT(latLngToCell)(&LatLng, i, &h3));
             char failureMessage[BUFF_SIZE];
             sprintf(failureMessage, "isValidCell failed on resolution %d", i);
             t_assert(H3_EXPORT(isValidCell)(h3), failureMessage);
         }
     }
 
     TEST(isValidCellDigits) {
-        GeoPoint GeoPoint = {0, 0};
+        LatLng LatLng = {0, 0};
```

Comment:
Nit: Not sure why we were naming the variable with this casing, but this one maybe should be fixed

Reply:
this looks like the variable was given that name by accident and we should adjust it

Reply:
updated